_includes/footnote.html

```diff
@@ -0,0 +1 @@
+<span class="footnote"><div class="footnoteborder"></div>{{ include.text }}</span>
```

Comment:
```suggestion
<span class="footnote"><div class="footnoteborder"></div>{{ include.text }}</span>

```

Reply:
Nope - if we do this, then after every one of these it will render a newline into the completed HTML